config.go

```diff
@@ -0,0 +1,280 @@
+			It is used to configure envbuilder dynamically during the runtime.
+			e.g. specifying whether to start ` + "`systemd`" + ` or ` + "`tiny init`" + ` for PID 1.`,
+		},
+		{
+			Name:        "Init Script",
+			Env:         "INIT_SCRIPT",
+			Default:     "sleep infinity",
+			Value:       serpent.StringOf(&c.InitScript),
+			Description: "InitScript is the script to run to initialize the workspace.",
+		},
+		{
+			Name:        "Init Command",
+			Env:         "INIT_COMMAND",
+			Default:     "/bin/sh",
+			Value:       serpent.StringOf(&c.InitCommand),
+			Description: "InitCommand is the command to run to initialize the workspace.",
+		},
+		{
+			Name:  "Init Args",
+			Env:   "INIT_ARGS",
+			Value: serpent.StringOf(&c.InitArgs),
+			Description: `InitArgs are the arguments to pass to the init command.
+			They are split according to ` + "`/bin/sh`" + ` rules with
+			https://github.com/kballard/go-shellquote.`,
+		},
+		{
+			Name:  "Cache Repo",
+			Env:   "CACHE_REPO",
+			Value: serpent.StringOf(&c.CacheRepo),
+			Description: `CacheRepo is the name of the container registry
+			to push the cache image to. If this is empty, the cache
+			will not be pushed.`,
+		},
+		{
+			Name:  "Base Image Cache Dir",
+			Env:   "BASE_IMAGE_CACHE_DIR",
+			Value: serpent.StringOf(&c.BaseImageCacheDir),
+			Description: `BaseImageCacheDir is the path to a directory where the base
+			image can be found. This should be a read-only directory
+			solely mounted for the purpose of caching the base image.`,
+		},
+		{
+			Name:  "Layer Cache Dir",
+			Env:   "LAYER_CACHE_DIR",
+			Value: serpent.StringOf(&c.LayerCacheDir),
+			Description: `LayerCacheDir is the path to a directory where built layers
+			will be stored. This spawns an in-memory registry to serve
+			the layers from.`,
+		},
+		{
+			Name:  "Devcontainer Dir",
+			Env:   "DEVCONTAINER_DIR",
+			Value: serpent.StringOf(&c.DevcontainerDir),
+			Description: `DevcontainerDir is a path to the folder containing
+			the devcontainer.json file that will be used to build the
+			workspace and can either be an absolute path or a path
+			relative to the workspace folder. If not provided, defaults to
+			` + "`.devcontainer`" + `.`,
+		},
+		{
+			Name:  "Devcontainer JSON Path",
+			Env:   "DEVCONTAINER_JSON_PATH",
+			Value: serpent.StringOf(&c.DevcontainerJSONPath),
+			Description: `DevcontainerJSONPath is a path to a devcontainer.json file
+			that is either an absolute path or a path relative to
+			DevcontainerDir. This can be used in cases where one wants
+			to substitute an edited devcontainer.json file for the one
+			that exists in the repo.`,
+		},
+		{
+			Name:  "Dockerfile Path",
+			Env:   "DOCKERFILE_PATH",
+			Value: serpent.StringOf(&c.DockerfilePath),
+			Description: `DockerfilePath is a relative path to the Dockerfile that
+			will be used to build the workspace. This is an alternative
+			to using a devcontainer that some might find simpler.`,
```

Comment:
I think Go will keep the break in the string so this will become: `...to the Dockerfile that			will be used to build the workspace.`

Just use a one-liner `Description: "..."`,

Reply:
Ahhhh, I see 🤔